targettest/sym1/symNotepad.c

```diff
@@ -0,0 +1,191 @@
+// --------------------------------------------------------------------------
+// Sym-1 Notepad
+//
+// Wayne Parham
+//
+// wayne@parhamdata.com
+// --------------------------------------------------------------------------
+//
+// Note:  This program requires RAM memory in locations 0xE000 - 0xEFFF
+//        Alternatively,  the tape I/O buffer location and size can  be
+//        changed by altering the defined TAPIO values below.
+//
+// --------------------------------------------------------------------------
+
+#include <sym1.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define     TAPIO_ADDRESS      0xE000
+#define     TAPIO_MAX_SIZE     0x0FFF
```

Comment:
Don't put content on fixed columns. Here:

```
#define TAPIO_ADDRESS  0xE000
#define TAPIO_MAX_SIZE 0x0FFF
```

Reply:
Updated.  Please check and confirm.